### ISSUE TITLE
Questions about theming

### ISSUE THREAD

**Issue by melck** [marp-team/marp-core#134]
Hello

I have a made a custom theme and now i want to have a theme that support multiple size (4:3 16:9 4K A4) with completely different css rules for size A4. 

Do you have a class applied when size is setup ?

**Comment by yhatt** [comment]
There is no applied class even if setting up `size` global directive at the moment. [Marpit local directives](https://marpit.marp.app/directives?id=local-directives-1) are assigned data attributes (`data-xxx="xxx"`) and CSS variables to `<section>` elements, but global directives are not.

As my position, I recommend to divide theme file if you want greatly different styling, to avoid too complex CSS.

But what your use case can we cover if Marp Core had assigned some class (or data attribute) depending on size?

**Comment by melck** [comment]
Ok, I understand. Firstly I use mainly marp to write slide deck. 

I like so much the simplicity that i wanted to use it to make printed document (A4 size). I need header, footer, page, etc. To keep this simple i wanted to include all css in one theme and split it by size. 

With this i will be able to write only markdown with vscode, and never have to use office product again :D.

Do you think marpit can be used for that ?

**Comment by yhatt** [comment]
I feel you. There are not many better solution for Markdown -> PDF document conversion with simple styling.

I'll consider to expose which size selected through global directive. In theme CSS, specific style could apply by size.

```css
/*!
 * @theme xxx
 * @size 4:3 960px 720px
 * @size A4 210mm 297mm
 */

section {
  /* Default style (16:9) */
}

section[data-size="4:3"] {
  /* Style for 4:3 slide */
}

section[data-size="A4"] {
  /* Style for A4 slide */
}
```

**Comment by melck** [comment]
Perfect it's exactly what i need :)

**Comment by yhatt** [comment]
`data-size` attribute is available in just released [v1.0.0](https://github.com/marp-team/marp-core/releases/v1.0.0).
